.github/workflows/deploy-docs.yaml

```diff
@@ -20,16 +20,16 @@ jobs:
         run: |
           curl -sL https://api.github.com/repos/weaveworks/pctl/releases/latest 2>/dev/null | \
           jq -r '.assets[] | select(.browser_download_url | contains("linux_amd64")) | .browser_download_url' | \
-          xargs curl -sL | tar xvz -C /tmp && chmod +x /tmp/pctl && /tmp/pctl --help
+          xargs curl -sL | tar xvz -C /usr/local/bin && chmod +x /usr/local/bin/pctl && pctl --help
       - name: Release to GitHub Pages
         env:
           USE_SSH: true
           GIT_USER: git
         run: |
+          make docgen
```

Comment:
does this mean the generated schemas aren't committed to `main`?

Reply:
reading further down it seems yes and thats intentional

Reply:
yes